town.logic

```diff
@@ -1,2 +1,104 @@
 area Town:
-  room Main
\ No newline at end of file
+  room Main:
+    node Anju:
+      chest AnjuHP
+    node AnjuRoof:
+      door AnjuChimney
+    node CatsMinish:
+      door Fountain
+      door StratoMinishBack
+    node FigurineHouseDoorEast:
+      door FigurineHouseEast
+    node GormanHouseSouth:
+      door GormanHouseSouth
+    node HearthLedge:
+      door HearthLedge
+      chest HearthLedge
+    node JuliettaRoof:
+      door JuliettaChimney
+      door RomioChimneyEast
+    node LibraryRoof:
+      door LibraryRoof
+    node LibraryRoofMinish:
+      door FlipperCave
+      door LibraryMinish
+    LibraryRoof -> LibraryRoofMinish: item Cane
+    node Main:
+      chest CafeLady
+      door Bakery
+      door Borlov
+      door East
+      door FigurineHouseWest
+      door HearthNorth
+      door HearthSouth
+      door GormanHouseWest
+      door Julietta
+      door LibraryBottom
+      door MayorWest
+      door North
+      door PostOffice
+      door Rem
+      door Romio
+      door SchoolWest
+      door Simon
+      door South
+      door Stockwell
+      door Strato
+      door Swiftblade
+      door Well
+      entrance Crest
+    Main <-> Anju: (item Flippers | item Cape)
+    Main <-> FigurineHouseDoorEast: item CarlovMedal
+    Main <-> GormanHouseSouth: flag TalkedToOracle
+    node MainBellHP:
+      chest MainBellHP
+    Main <-> MainBellHP: item Cape 
+    node MainMinish:
+      door AnjuMinish
+      door BakeryMinish
+      door CafeMinish
+      door CarpenterBuildingMinish
+      door DrLeftMinish
+      door HearthMinish
+      door RomeoMinish
+    node MayorGarden:
+      door MayorEast
+      door MayorBasementLadder
+    node MinishBridge:
+      door CafeBridgeMinish
+      door StockwellBridgeMinish
+    node RomioRoof:
+      door RomioChimneyWest
+    node SchoolGarden:
+      door SchoolEast
+    node SchoolGardenMinish:
+      door SchoolEntranceMinishEast
+      door SchoolGarden
+    node SchoolRoof:
+      chest SchoolRoof
+      door SchoolEntranceMinishWest
+    node SouthWest:
+      door Carlov
+      door Carpenters
+      door DrLeft
+    Main <-> SouthWest: (item Flippers | item Cape | flag DogMoves)
+    node TownDigging:
+      door TownDigging
+    Main <-> TownDigging: item MoleMitts
+    node Waterfall:
+      door Waterfall
+    Main <-> Waterfall: (item Flippers | flag TownWaterfallFusion)
```

Comment:
This needs to be &
```suggestion
    Main <-> Waterfall: (item Flippers & flag TownWaterfallFusion)
```
And I think you only need the fusion to access the entrance, not to get away from the exit (which would only require flippers)?